automatic/cutepdf/tools/chocolateyInstall.ps1

```diff
@@ -1,16 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
 $checksum = '2D9E07F25358C9D2317BC639AFCDEDDB893D1FCFD43BB66FF372DBA11E169EE1'
 $url = 'http://www.cutepdf.com/download/CuteWriter.exe'
 
 $packageArgs = @{
   packageName   = 'cutepdf'
-  unzipLocation = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
   fileType      = 'exe'
   url           = $url
   silentArgs    = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-' # Inno Setup Package
   validExitCodes= @(0)
-  softwareName  = 'cutepdf*'
```

Comment:
Don't remove software name, this is used in pro/business version of choco to detect if it's already installed.